static/scripts/teams.js

```diff
@@ -196,6 +197,25 @@ $(document).ready(() => {
 			payload: { userId },
 		});
 
+		const $deleteInfoAlert = $leaveTeamModal.find('.alert');
+		const $fileList = $leaveTeamModal.find('.file-list');
+
+		if (files && files.length > 0) {
```

Comment:
U can simplify with `if (files?.length) {}`

Reply:
Changed according to suggestions